packages/contracts/ts/scripts/getBlueprint.ts

```diff
@@ -0,0 +1,96 @@
+import fs from "fs";
+import path from "path";
+
+interface BlueprintEntry {
+  coord: [number, number, number];
+  id: number;
+  orientation: number;
+}
+
+interface IndexerResponse {
+  block_height: number;
+  result: Array<Array<string[]>>; // Nested array with string values
+}
+
+async function queryIndexer(sql: string): Promise<IndexerResponse> {
+  const INDEXER_URL = process.env.INDEXER_URL || "https://dozer.onrender.com/q";
```

Comment:
we're gonna close this indexer actually so should switch to the one defined on the redstone chain obj from mud pkg